chain/chain/src/runtime/mod.rs

```diff
@@ -720,8 +726,19 @@ impl RuntimeAdapter for NightshadeRuntime {
         // Total amount of gas burnt for converting transactions towards receipts.
         let mut total_gas_burnt = 0;
         let mut total_size = 0u64;
-        // TODO: Update gas limit for transactions
-        let transactions_gas_limit = gas_limit / 2;
+
+        let transactions_gas_limit = if checked_feature!(
+            "protocol_feature_nep539_congestion_control",
+            Nep539CongestionControl,
+            protocol_version
+        ) {
```

Comment:
Well, that's how is used to be used^^. You start with the feature name while it's not stabilized, which uses cargo features to only compile the code if the feature is enabled. In the stabilization PR, you put in "stable" and ideally don't need to touch anything else in the code snippet.

I don't know why you would use the `checked_feature` macro at all if you don't want to make use of cargo features. You might as well check `protocol_version >= Nep539CongestionControl.protocol_version()`, right?

But if the current practise is to avoid cargo features, we can certainly do it that way. We just need to be a bit more careful and might have to add a bunch of ignore warning attributes.